app/src/main/java/com/neo/speaktouch/service/SpeakTouchService.kt

```diff
@@ -54,6 +57,16 @@ class SpeakTouchService : AccessibilityService() {
         )
     }
 
+    override fun onServiceConnected() {
+        super.onServiceConnected()
+
+    if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.S) {
+            val info = serviceInfo
+            info.flags = info.flags or AccessibilityServiceInfo.FLAG_REQUEST_2_FINGER_PASSTHROUGH
```

Comment:
What is this flag really for, I tested it with and without it on Android 12 and didn't notice a difference.

Reply:
1. No XML representation for this, unfortunately.
2. This flag is really needed because you can't e.g. swipe with two fingers without this flag. Tested with and without. All two fingers default gestures are affected. You cannot unlock the screen, you cannot open list of apps by swiping two fingers up on the home screen, etc.